README.md

```diff
@@ -1,4 +1,49 @@
-# Spotify Playlist Generator
+![screenshot](https://github.com/Harry-Lees/Spotify-Playlist-Generator1/blob/master/.github/screenshot.png) 
+
+<!-- TABLE OF CONTENTS -->
+## Table of Contents
+
+* [About](#about-the-project)
+  * [Built With](#built-with)
+* [Contributing](#contributing)
+* [License](#license)
+
+## About The Project
+
+### Spotify Playlist Generator
 
 A Flask web app to generate a Spotify playlist based on selected tracks and personal preferences.
 
+### Built With
+This section should list any major frameworks that you built your project using. Leave any add-ons/plugins for the acknowledgements section. Here are a few examples.
+* [Bootstrap](https://getbootstrap.com)
+* [Flask](https://jquery.com)
```

Comment:
Is the Flask link supposed to link to jQuery's website?

Reply:
No, thank you for spotting that. I've updated it now.